### PR TITLE
[OSDEV-1144] Claims emails. Updated text for approval and denied emails.

### PR DESCRIPTION
**Pull request by Innavin369**
[OSDEV-1144](https://opensupplyhub.atlassian.net/browse/OSDEV-1144) Claims emails. Updated text for approval, revocation, and denial emails.

[OSDEV-1144]: https://opensupplyhub.atlassian.net/browse/OSDEV-1144?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
image
image
image

### CODE REVIEW

**Review by mazursasha1990 — CHANGES_REQUESTED** [review]
Please, add the requested changes and fix the linter error in pipelines.

File: src/django/api/templates/mail/data_team_signature_block.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <p>
-    OS Hub Data Team
+    The OS Hub Claims Team
 </p>
```
[the remark this answers is not in the document]
Comment (Innavin369) [reply src/django/api/templates/mail/data_team_signature_block.html:3]: changed back

File: src/django/api/templates/mail/data_team_signature_block.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {% block content %}
-OS Hub Data Team
+The OS Hub Claims Team
 {% endblock content %}
```
[the remark this answers is not in the document]
Comment (Innavin369) [reply src/django/api/templates/mail/data_team_signature_block.txt:3]: changed back

File: src/django/api/templates/mail/claim_facility_revocation_body.txt [hunk 3]
```diff
@@ -1,18 +1,23 @@
...
 
-You're receiving this email because we revoked your request to claim this Open Supply Hub facility:
+You are receiving this email because your request to claim the following production location on
+Open Supply Hub has been revoked:
 
-- Facility: {{ facility_name }}, {{ facility_address }}, {{ facility_country }}
-- Facility URL: {{ facility_url }}
+- Name of Production Location: {{ production_location_name }}
+- Address of Production Location: {{ production_location_address }}, {{ production_location_country }}
+-  OS Hub Profile: {{ production_location_url }}
```
[the remark this answers is not in the document]
Comment (Innavin369) [reply src/django/api/templates/mail/claim_facility_revocation_body.txt:9]: done

**Review by mazursasha1990 — CHANGES_REQUESTED** [review]
Please use `claims_team_signature_block` for each email template you have changed, without `data_team_signature_block`. 
Additionally, according to the acceptance criteria, you should slightly modify the text in the `claims_team_signature_block`. Currently, the text is:

> OS Hub Claims Team

**Review by roman-stolar — DISMISSED** [review]
LGTM

**Review by mazursasha1990 — CHANGES_REQUESTED** [review]
Please add the requested changes https://github.com/opensupplyhub/open-supply-hub/pull/304#pullrequestreview-2207451498

**Review by mazursasha1990 — APPROVED** [review]
LGTM

**Review by roman-stolar — APPROVED** [review]
Approved

**Pull request merged**